.github/ISSUE_TEMPLATE/RELEASE.md

```diff
@@ -0,0 +1,20 @@
+## Update Code
+- [x] #issue that needs to be picked up
+- [x] #PR that needs to be reviewed
+
+## Update source data
+### Partner Agency Data
+- [x] all parterner agency source data updated #tag the issue template from data repo here
```

Comment:
This sentence doesn't make sense to me

Reply:
Hey @mbh329  think it is time to revisit this PR. I wonder if this makes more sense now we have gone through the source data updating process. I basically want to link the master issue from data repo issue here. Like [this one](https://github.com/NYCPlanning/db-knownprojects-data/issues/174)

Reply:
I think the template needs to be improved, happy to help you with this. I don't think we typically delineate between _dcp data_ and _geographies data_  like we do in this one. dcp_boroboundaries_wi is in here twice and dcp_mappluto_wi could be interpreted as a geography so my first recommendation would be to combine these two

Reply:
@mbh329 removed the geography. Let me know if it looks good. Thanks